sci-physics/geant-vmc/geant-vmc-4.3.6.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils versionator
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI=" http://root.cern.ch/git/geant4_vmc.git"
+else
+	DOWN_PV=$(get_version_component_range 2-)
+	SRC_URI="http://root.cern.ch/download/vmc/geant4_vmc.${DOWN_PV}.tar.gz"
+	SOURCE_PV=$(replace_all_version_separators - ${DOWN_PV})
+	S="${WORKDIR}/geant4_vmc-${SOURCE_PV}"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="Virtual Monte Carlo Geant4 implementation"
+HOMEPAGE="http://root.cern.ch/root/vmc/VirtualMC.html"
+
+LICENSE="GPL-2"
+SLOT="4"
```

Comment:
Given this is a new package, any reason to use non-zero slot? Do you expect to have future versions that will be co-installable in parallel?

Reply:
@mgorny : This package is directly bound to `sci-physics/geant` and its `SLOT`s. 
Right now, there is only `sci-physics/geant:4` in tree (still the slot was kept).

In the past, there also was `sci-physics/geant:3` in-tree and a co-installable `sci-physics/geant-vmc:3` was living in the gentoo-science-overlay. 

Of course, it's hard to tell the future, but it is likely that GeantV ( http://geant.cern.ch/ ), whenever it sees some stable release will need a separate VMC interfacing library. 

I have no very strong opinion, but currently I'd tend to stay close to the `SLOT` choice also employed for `sci-physics/geant`. What do you think?